tests/test_equivalence.py

```diff
@@ -20,13 +21,43 @@
 from fasm2bels.fasm2bels import main
 
 
-test_names = ["add32"]
+test_names = ["add32","alu"]
 
 
 def unpack_tar(tar_file):
     tar = tarfile.open(name=tar_file, mode="r:gz")
     tar.extractall(path=os.path.dirname(tar_file))
 
+def createGoldenfile(fileName):
+    path = os.environ['VIVADO_PATH'] #Note: Please use export VIVADO_PATH=/tools/Xilinx/Vivado/(VERSION OF VIVADO YOU'RE USING)/bin/vivado before running this.
```

Comment:
You may add something like the following to make sure that `VIVADO_PATH` is set:

```
vivado = os.environ.get('VIVADO_PATH', None)

assert vivado is not None, "VIVADO_PATH environmental variable was not set!"
```